var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -97,6 +98,12 @@ def make_defs(self):
             'FC={0}'.format(spack_f77),
             'MAKE_NO_J=1'
         ]
+        # invoke make with the correct TARGET for aarch64
+        if 'aarch64' in spack.architecture.sys_type():
+            make_defs += [
+                'TARGET=PILEDRIVER',
+                'TARGET=ARMV8'
+        ]
```

Comment:
You went one level too far. The closing bracket needs to be indented the same as `make_defs`.